### PR TITLE
feat: Use univeral bootstrap for starting the system.

### PR DESCRIPTION
**Pull request by galthaus**

### CODE REVIEW

File: cleanup.sh [hunk 1]
```diff
@@ -80,6 +80,7 @@ do
   then
     drpcli machines wait Name:$mc WorkflowComplete true 120
     drpcli machines destroy Name:$mc
+    drpcli endpoints destroy $mc >/dev/null 2>/dev/null || :
```
Comment (zehicle) [inline cleanup.sh:83]: the site-destroy does does this - I wanted it inside the workflow because I'll setup sites multiple times.

Comment (zehicle) [reply cleanup.sh:83]: does no harm, so OK to leave in

Comment (galthaus) [reply cleanup.sh:83]: Yes - but if things fail - that doesn't run so this is clean that lets things exit later.

**Comment by zehicle** [comment]
:+1:

**Pull request merged**